server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -1960,9 +1960,6 @@ func (s *BuildBuddyServer) serveBytestream(ctx context.Context, w http.ResponseW
 		return http.StatusBadRequest, err
 	}
 
-	// TODO(siggisim): Figure out why this JWT is overriding authority auth and remove.
-	ctx = context.WithValue(ctx, "x-buildbuddy-jwt", nil)
```

Comment:
:o

Reply:
I split this out into https://github.com/buildbuddy-io/buildbuddy/pull/8115